skimage/filters/thresholding.py

```diff
@@ -902,10 +902,17 @@ def threshold_niblack(image, window_size=15, k=0.2):
     -----
     This algorithm is originally designed for text recognition.
 
+    The Bradley threshold is a particular case of the Niblack
+    one, being equivalent to
+    >>> threshold_niblack(image, k=0) * (1-q)
```

Comment:
(a) I think the syntax might require some spacing for proper formatting in the docs:

```suggestion

    >>> q = 0.95
    >>> t_brad = threshold_niblack(image, k=0) * (1-q)

```

(b) I used quantile in my explanation because I think they are to percentiles what radians are to degrees, but based on your previous PR I imagine they use percentiles. Therefore I presume they used q = 1. And further, no, they must have used q = 0, because otherwise it reduces to 0 everywhere. (ie they used (1-q) = 1.

Other than those two comments, the PR looks good to me, thanks! =)

Reply:
Hey Juan,
now I'm kinda confused. :sweat_smile: 
Yep, they use percentiles. Their algorithm shows (page 4):
```
20: if (in[i, j) * count) <= (sum * (100-t)/100) then
```
Then Bradley's should be equal to `threshold_niblack()` with `k=0` only, as before? :smile:

Reply:
Well if t = 100, that number is equal to 0! So someone made a mistake somewhere. =)

Reply:
Also, `(100-t)/100` is equal to `(1-q)` where `q=t/100`, ie quantile is the percentile divided by 100.

Reply:
> someone made a mistake somewhere. =)

It was me, of course. :smile: Let's try to rewrite this then...